pypet2bids/tests/test_helper_functions.py

```diff
@@ -177,7 +177,7 @@ def test_get_recon_method():
             "contents": "PSF+TOF 3i21s",
             "subsets": 21,
             "iterations": 3,
-            "ReconMethodName": "PSF+TOF",
+            "ReconMethodName": "PSF+TOF3i21s",
```

Comment:
Time of Flight as a recon method does not have iteration and subsets -- not a useful test case